USAGE.md

```diff
@@ -31,16 +31,28 @@ To use the theme JSON file in your app, follow the steps below:
         - assets/appainter_theme.json
     ```
 
-4. Update your `main` function to load your theme. 
+4. Update your `main` function to be async, load your theme and pass it into your App
 
     ```dart
-    void main() async {
-      WidgetsFlutterBinding.ensureInitialized();
-      final themeStr = await rootBundle.loadString('assets/appainter_theme.json');
-      final themeJson = jsonDecode(themeStr);
-      final theme = ThemeDecoder.decodeThemeData(themeJson)!;
-    }
+      void main() async {
+        WidgetsFlutterBinding.ensureInitialized();
+
+        final themeStr = await rootBundle.loadString('assets/appainter_theme.json');
+        final themeJson = jsonDecode(themeStr);
+        final theme = ThemeDecoder.decodeThemeData(themeJson)!;
+
+        runApp(MyApp(theme: theme));
+      }
+
+      class MyApp extends StatelessWidget {
+        final ThemeData theme;
+
+        const MyApp({Key? key, required this.theme}) : super(key: key);
+        ....
     ```
+    Tips for those new to flutter/dart:
+    - `rootBundle` is available by importing `'package:flutter/services.dart';`
+    - `jsonDecode` is available by importing `'dart:convert';`
 
 5. You can then pass in `theme` to your `MaterialApp` widget to use it.
```

Comment:
Can you also update the `MyApp` widget that you included in the snippet above and complete the widget by making it to return the following app/widget? After that you can also remove this step.

```dart
MaterialApp(
  theme: theme,
  title: 'Appainter',
  home: HomePage(),
);
```